### DISCUSSION TITLE
Key binding for interactive rebase?

### DISCUSSION THREAD

**Discussion by v0d1ch** [NeogitOrg/neogit#507] [Q&A] [answered]
When I do interactive rebase onto current branch I am presented with a log of commits. I am not sure how to pick/squash commits from this list - what are the key bindings I can use for this?

**Comment by CKolkey** [comment, marked as answer]
It's actually just a normal `gitrebase` buffer, so there aren't (yet) any neogit-specific keybindings for that buffer. Just edit it like normal :)

**Reply by v0d1ch** [reply]
Thanks!
